src/scripts/dungeons/Dungeon.ts

```diff
@@ -7,6 +7,7 @@ class Dungeon {
 
     name: KnockoutObservable<string> = ko.observable("");
     pokemonList: string[];
+    itemList: string[]
```

Comment:
@tklaxbro put a semicolon here to make me happy.

Reply:
Why does this still show up as a string[]?